lib/Config/Menu.php

```diff
@@ -836,6 +837,13 @@ function checkPremiumKey(ServicesChecker $checker = null) {
     $this->premium_key_valid = $checker->isPremiumKeyValid($show_notices);
   }
 
+  function checkFromEmailAuthorization(ServicesChecker $checker = null) {
```

Comment:
Why is this function public? and why does it have an optional argument if you never pass it in?

Reply:
Consistency. Because above this, there are two almost identical checks written the completely same way. If I wrote this one differently (even though it would be more correct), in the future it would make less sense why are they written differently. So I copied the style of those functions because someone once wrote it that way.

Applied also for next comment.

Reply:
But the other functions are actually using the argument. And they were written before we decided to start using DI.

Reply:
Didn't notice it, that they are used in tests. Privatized and argument removed.